### ISSUE TITLE
Only pokestops mode?

### ISSUE THREAD

**Issue by Agora0129** [jabbink/PokemonGoBot#502]
Is there a way you can change it so that it only goes to pokestops? Reason I'm asking is when I run out of poke balls I get a No ball error, and in order to prevent that from happening I go to the pokestops.

**Comment by shilch** [comment]
Yes, turn of catching pokemon

**Comment by Agora0129** [comment]
Is that in the config.properties? I don't see it.

**Comment by Agronis** [comment]
If you're using 4.3 it is.

# Should the bot catch pokemons (true/false)
catch_pokemon=true

**Comment by shilch** [comment]
```
# Should the bot catch pokemons (true/false)
catch_pokemon=true
```

**Comment by Agora0129** [comment]
I'm still on 4.0. Im glad that's there, thanks for the help.

**Comment by shilch** [comment]
Did you turn on pokestops

**Comment by Agora0129** [comment]
Yes.

# Should the bot loot the pokestops (true/false)
loot_pokestop=true

**Comment by fongoid** [comment]
Looks like this was a bug, but is now fixed (in next release after 0.4.3) https://github.com/jabbink/PokemonGoBot/commit/95eecdf47660eb39c600e8d0d9385ed94b012b8c
